web-admin/src/features/organizations/hibernating/OrganizationHibernatingForAdmins.svelte

```diff
@@ -0,0 +1,102 @@
+  import CTAHeader from "@rilldata/web-common/components/calls-to-action/CTAHeader.svelte";
+  import CTAMessage from "@rilldata/web-common/components/calls-to-action/CTAMessage.svelte";
+  import CTANeedHelp from "@rilldata/web-common/components/calls-to-action/CTANeedHelp.svelte";
+  import AlertCircleIcon from "@rilldata/web-common/components/icons/AlertCircleOutline.svelte";
+  import CheckCircleOutline from "@rilldata/web-common/components/icons/CheckCircleOutline.svelte";
+  import LoadingCircleOutline from "@rilldata/web-common/components/icons/LoadingCircleOutline.svelte";
+  import MoonCircleOutline from "@rilldata/web-common/components/icons/MoonCircleOutline.svelte";
+
+  export let organization: string;
+
+  $: billingIssueMessage = useBillingIssueMessage(organization);
+  $: bannerCTAHandler = new BannerCTAHandler(organization);
+  $: ({
+    showStartTeamPlanDialog,
+    startTeamPlanType,
+    teamPlanEndDate,
+    wakingProjects,
+  } = bannerCTAHandler);
+  let issueForHibernation: BillingIssueMessage;
+  $: if ($billingIssueMessage.data) {
+    if ($billingIssueMessage.data.type === "error") {
+      // only show the hibernating message if it is an error
+      issueForHibernation = $billingIssueMessage.data;
+    } else {
+      // else who a CTA for waking up projects
+      issueForHibernation = {
+        type: "default",
+        iconType: "sleep",
+        title: "Your projects are hibernating",
+        description: "",
+        cta: {
+          text: "Wake projects",
+          type: "wake-projects",
+        },
+      };
+    }
+  }
+
+  const IconMap = {
+    alert: AlertCircleIcon,
+    check: CheckCircleOutline,
+    sleep: MoonCircleOutline,
+    loading: LoadingCircleOutline,
+  };
+</script>
+
+{#if $wakingProjects}
+  <div class="flex flex-col justify-center items-center gap-y-6 my-20">
+    <div class="flex flex-col gap-y-2">
+      <LoadingCircleOutline size="104px" className="text-slate-300" />
+      <CTAHeader>Hang tight! We're waking up your projects...</CTAHeader>
+      <CTANeedHelp />
+    </div>
+  </div>
+{:else if issueForHibernation}
+  <div class="flex flex-col justify-center items-center gap-y-6 my-20">
+    <div class="flex flex-col gap-y-2">
+      {#if issueForHibernation.iconType in IconMap}
+        <!-- TODO: gradient -->
+        <svelte:component
+          this={IconMap[issueForHibernation.iconType]}
+          size="104px"
+          className="text-slate-300"
+          gradientEnd="slate-200"
+        />
+      {/if}
+      <CTAHeader variant="bold">
+        {issueForHibernation.title.replace(/\.$/, "")}
+      </CTAHeader>
+      {#if issueForHibernation.description}
+        <CTAMessage>
+          {issueForHibernation.description.replace(/\.$/, "")}
+        </CTAMessage>
+      {/if}
+    </div>
+    {#if issueForHibernation.cta}
+      <Button
+        type="secondary"
+        wide
+        on:click={() => bannerCTAHandler.handle(issueForHibernation)}
```

Comment:
This isn't a banner, so seems like `bannerCTAHandler` needs a new name? `billingCTAHandler`?

Reply:
Agreed. Renamed the class itself. It was added when these hibernation updates was not in the mocks.